src/Symfony/Component/Validator/Mapping/PropertyMetadata.php

```diff
@@ -48,7 +48,15 @@ public function __construct($class, $name)
      */
     public function getPropertyValue($object)
     {
-        return $this->getReflectionMember($object)->getValue($object);
+        $reflProperty = $this->getReflectionMember($object);
+
+        $checkPropertyInitialization = \PHP_VERSION_ID >= 70400;
```

Comment:
No need for the variable, should be done inline.

Reply:
Done